packages/eslint-config-airbnb/CHANGELOG.md

```diff
@@ -1,132 +1,172 @@
 6.0.2 / 2016-02-22
 ==================
-- [fix] disable `no-confusing-arrow` due to an `eslint` bug (#752)
+- [fix] disable [`no-confusing-arrow`][no-confusing-arrow] due to an `eslint` bug ([#752](https://github.com/airbnb/javascript/issues/752))
```

Comment:
Oh, no real reason, just for consistency – since 6.0.0 there was no space anymore and I thought that might be the new style, so I adjusted the rest.